### ISSUE TITLE
Closing labels can be missleading when there are nested calls

### ISSUE THREAD

**Issue by FMorschel** [Dart-Code/Dart-Code#5368]
**Is your feature request related to a problem? Please describe.**
Closing labels can be misleading when you have nested calls (see the screenshot below).

**Describe the solution you'd like**
Not sure what is best here. Any other suggestions are appreciated.

- Disable
- Show it before the `.` for the call

Show the new type doesn't seem like the right call since the goal of this is to make you find the related starting line.

**Additional context**
![image](https://github.com/user-attachments/assets/869500cd-d913-4117-bbc0-a44abc6c9dc4)

**Comment by DanTup** [comment]
Yeah, it's not ideal that the `<int>[]` label is so far away from the `]`.

I'm not sure putting it next to the `]` will be good though - the original prototype did that, but it got very confusing if you had multiple closes on the same page:

```
) // row ) // column // container
```

It wasn't super obvious what was code and what was fake comments, so we merge them and put them at the end.

We could remove them, but I wonder if there's still value in having them like this, because you can still see that the first closing element in the line is the array.

I'd be interested in others thoughts/opinions.
